src/views/v2/commerce/checkout/purchaser-info.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Purchaser Info.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/checkout/purchaser-info.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ *
+ * @var \Tribe__Template $this [Global] Template object.
+ * @var bool $is_logged_out [Global] If the user is logged out.
+ * @var bool $must_login Global] Whether login is required to buy tickets or not.
+ */
+
+if ( ! $is_logged_out || $must_login ) {
```

Comment:
Let's use the template tag for checking if the user is logged in. 

We should probably also bail if there are no items in cart, as the checkout page can be accessed at all time.